base/src/main/java/com/smartdevicelink/proxy/rpc/UnsubscribeVehicleData.java

```diff
@@ -505,22 +527,28 @@ public Boolean getInstantFuelConsumption() {
     }
 
     /**
-     * Sets a boolean value. If true, unsubscribes from externalTemperature data
+     * Sets the externalTemperature.
      *
-     * @param externalTemperature a boolean value
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.

```

Comment:
@kostyaBoss On line 533, I got 7.1.0 instead of 7.1 using the RPC generator tool. I would replace the 7.1 with 7.1.0.